runner/src/main/scala/stress/utils/Converters.scala

```diff
@@ -0,0 +1,5 @@
+package stress.utils
+
+object Converters {
```

Comment:
Don't we have an `Ids` service in XLR that does something like this? So maybe rename to be inline with XLR?